.github/workflows/release-promotion-tests.yml

```diff
@@ -175,6 +175,19 @@ jobs:
           author_name: "Workspace Manager ${{ steps.set-env-step.outputs.test-env }} integrationTest"
           fields: repo,job,workflow,commit,eventName,author,took
 
+      - name: "Notify Workspaces test failure Slack"
+        if: failure()
```

Comment:
Add to notify on failure to our channel. There is a previous step to notify to #dsde-qa on promotion to alpha and staging.